src/averageAges.js

```diff
@@ -14,12 +14,29 @@
  *
  * @return {number}
  */
+
+function getAverage(totalAgeSum, length) {
+  return +(totalAgeSum / length).toFixed(2);
+}
+
 function calculateMenAverageAge(people, century) {
   // write code here
   // learn how to use array methods like .filter .map .some .every .find .reduce
   // avoid using loop and forEach
   // replace `if ()` statement with &&, || or ?:
   // without nesting
+  const filteredPeople = people.filter(human =>
+    century !== undefined
+      ? human.sex === 'm' && century === Math.ceil(human.died / 100)
+      : human.sex === 'm');
+
+  ;
+
+  const totalAgeCentury = filteredPeople.reduce(
+    (a, b) => a + (b.died - b.born), 0
+  );
```

Comment:
you use it twice, so you can move it in the separate function

Reply:
it is done in new commit )